x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/pipelines.tsx

```diff
@@ -14,9 +14,10 @@ import { i18n } from '@kbn/i18n';
 import { DataPanel } from '../../../../shared/data_panel/data_panel';
 
 import { InferencePipelineCard } from './inference_pipeline_card';
+import { IngestPipelinesCard } from './ingest_pipelines_card';
 import { InferencePipeline } from './types';
 
-export const SearchIndexPipelines: React.FC = () => {
+export const SearchIndexPipelines: React.FC<{ mlPipelinesEnabled: boolean }> = () => {
   // TODO: REPLACE THIS DATA WITH REAL DATA
```

Comment:
Is this TODO still relevant?

Reply:
I think so, this is ML inference stuff not ours